controllers/repository_controller.go

```diff
@@ -108,10 +108,13 @@ func (r *RepositoryReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 	_ctx, _cancel := context.WithCancel(ctx)
 	r.C[key] = repository.NewChartmuseum(_ctx, logger, r.Client, r.Scheme, repo, _cancel)
-	r.C[key].Start()
+	if err := r.C[key].Start(); err != nil {
+		r.Recorder.Event(repo, v1.EventTypeWarning, "StartFail ", fmt.Sprintf("start %s fail", key))
```

Comment:
I think the `error` should be printed here to make it easier to troubleshoot the problem.

Reply:
pls remove extra blank in `StartFail `

Reply:
That will repeatedly output error because the error has been printed in func start().